tests/panels-features/panels-features-grid.spec.js

```diff
@@ -340,4 +340,17 @@ mainTest.describe(() => {
     await mainPage.waitForChangeIsSaved();
     await expect(mainPage.createdLayer).toHaveScreenshot('board-without-grid.png');
   });
+
+  mainTest(qase(1864,'Duplicate board with guide'), async ({ page, browserName }) => {
+    const mainPage = new MainPage(page);
+    const designPanelPage = new DesignPanelPage(page);
+    await designPanelPage.clickAddGridButton();
+    await mainPage.waitForChangeIsSaved();
+    await mainPage.clickShortcutCtrlD(browserName);
+    await mainPage.waitForChangeIsUnsaved();
+    await mainPage.waitForChangeIsSaved();
+    await expect(mainPage.viewport).toHaveScreenshot('two-board-with-guide.png', {
```

Comment:
Idem last comment

Reply:
We could forgo this screenshot, and alternatively, verify that two boards exist in the layout panel. However, this approach makes it difficult to ensure that the duplicate looks exactly the same as the copied object.

Reply:
Ok!